irods/helpers/__init__.py

```diff
@@ -0,0 +1,50 @@
+import functools
+import inspect
+import types
+import contextlib
+from ..test.helpers import make_session as make_test_session
+from irods.message import (ET, XML_Parser_Type)
+
+
+# See https://stackoverflow.com/questions/67028108/overriding-function-signature-in-help-when-using-functools-wraps
+def update_func(f, func_name='', update_kwargs: dict = None):
+    """Based on http://stackoverflow.com/a/6528148/190597 (Glenn Maynard)"""
+    g = types.FunctionType(
+            code=f.__code__,
+            globals=f.__globals__.copy(),
+            name=f.__name__,
+            argdefs=f.__defaults__,
+            closure=f.__closure__
+    )
+
+    g = functools.update_wrapper(g, f)
+    g.__signature__ = inspect.signature(g)
+    g.__kwdefaults__ = (f.__kwdefaults__ or {}).copy()
+
+    # Adjust your arguments
+    for key, value in (update_kwargs or {}).items():
+        g.__kwdefaults__[key] = value
+        g.__signature__.parameters[key]._default = value
+
+    g.__name__ = func_name or g.__name__
+    return g
+
+
+make_session = update_func(make_test_session,
+                           update_kwargs = dict(test_server_version = False))
```

Comment:
Add a comment explaining the motivation for this.

Reply:
done!